### PR TITLE
Sc 3314/amazon s3 move file s

### PR DESCRIPTION
**Pull request by Shamwayne**

### CODE REVIEW

**Comment by smjohnsonShipyard** [comment]
image
Getting this error on run time on my end.

File: amazons3_blueprints/move_file.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
+import os
+import boto3
+import botocore
+from botocore.client import Config
+import re
+import argparse
+import glob
+from ast import literal_eval
```
Comment (ericelsken) [inline amazons3_blueprints/move_file.py:8]: This isn't being used anywhere - you can remove?

File: amazons3_blueprints/move_file.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    return
+
+
+def s3_list_files(
+        s3_connection,
+        bucket_name,
+        source_folder,
+        ):
+    """List files in s3"""
+    s3_response = s3_connection.list_objects_v2(Bucket=bucket_name, Prefix=source_folder)
```
Comment (ericelsken) [inline amazons3_blueprints/move_file.py:95]: Is this guaranteed to return all matches, or is the call paged in some way. If it is paged, we need to update to fetch all pages.

**Review by ericelsken — CHANGES_REQUESTED** [review]

File: amazons3_blueprints/move_file.py (hunk 2 above)
Comment (Shamwayne) [reply amazons3_blueprints/move_file.py:95]: the maximum number of objects listed at a time is 1000 objects, with an option to start listing objects after an offset i.e (start listing from what would have been the 1000th matched object going onwards).

**Pull request merged**